web/public/locales/no-NB.json

```
@@ -1352,6 +1443,22 @@
     "SE": {
       "zoneName": "Sverige"
     },
+    "SE-SE1": {
+      "countryName": "Sverige",
+      "zoneName": "Norra Sverige (Luleå)"
+    },
+    "SE-SE2": {
+      "countryName": "Sverige",
+      "zoneName": "Norra Mellansverige (Sundsvall)"
+    },
+    "SE-SE3": {
+      "countryName": "Sverige",
+      "zoneName": "Södra Mellansverige (Stockholm)"
+    },
+    "SE-SE4": {
+      "countryName": "Sverige",
+      "zoneName": "Södra Sverige (Malmö)"
+    },
```

```suggestion
    "SE-SE1": {
      "countryName": "Sverige",
      "zoneName": "Norra Sverige"
    },
    "SE-SE2": {
      "countryName": "Sverige",
      "zoneName": "Norra Mellansverige"
    },
    "SE-SE3": {
      "countryName": "Sverige",
      "zoneName": "Södra Mellansverige"
    },
    "SE-SE4": {
      "countryName": "Sverige",
      "zoneName": "Södra Sverige"
    },
```
Since we don't use these "aliases" for the Swedish bidding zones in either the English or Swedish translation I would skip them here as well to avoid them getting truncated in the app.